src/sentry/static/sentry/app/views/settings/project/projectFilters/projectFiltersSettings.tsx

```diff
@@ -267,19 +270,23 @@ class ProjectFiltersSettings extends AsyncComponent<Props, State> {
               disabled={disabled || !hasFeature}
             />
           ))}
+
+          {hasFeature && this.props.project.options?.['filters:error_messages'] && (
+            <PanelAlert type="warning" data-test-id="error-message-disclaimer">
+              {t(
+                "Minidumps, errors in the minified production build of React, and Internet Explorer's i18n errors cannot by filtered by message."
```

Comment:
@jan-auer  Is this copy correct or would you change something?

Reply:
This is A+